project-base/app/src/DataFixtures/Demo/InquiryDataFixture.php

```diff
@@ -0,0 +1,485 @@
+        $inquiryData->companyName = 'Fragile Express';
+        $inquiryData->companyNumber = '66778899';
+        $inquiryData->companyTaxNumber = 'US66778899';
+        $inquiryData->note = 'Can you provide more information about the product line?';
+        $inquiryData->product = $inquiryProduct;
+        $inquiryData->createdAt = new DateTimeImmutable('2024-09-02 9:22:14');
+        $this->inquiryFacade->create($inquiryData);
+
+        $inquiryData = $this->inquiryDataFactory->create();
+        $inquiryData->firstName = 'Gabriel';
+        $inquiryData->lastName = 'Griffin';
+        $inquiryData->email = 'gabriel.griffin@example.com';
+        $inquiryData->telephone = '999444333';
+        $inquiryData->companyName = null;
+        $inquiryData->companyNumber = null;
+        $inquiryData->companyTaxNumber = null;
+        $inquiryData->note = 'Do you offer products for individual use?';
+        $inquiryData->product = $inquiryProduct;
+        $inquiryData->createdAt = new DateTimeImmutable('2024-09-02 10:15:41');
+        $this->inquiryFacade->create($inquiryData);
+
+        $inquiryData = $this->inquiryDataFactory->create();
+        $inquiryData->firstName = 'Sophia';
+        $inquiryData->lastName = 'Evans';
+        $inquiryData->email = 'sophia.evans@example.com';
+        $inquiryData->telephone = '555222888';
+        $inquiryData->companyName = 'Abstergo Industries';
+        $inquiryData->companyNumber = '44556677';
+        $inquiryData->companyTaxNumber = 'US44556677';
+        $inquiryData->note = 'Interested in more details on your memory research products.';
+        $inquiryData->product = $inquiryProduct;
+        $inquiryData->createdAt = new DateTimeImmutable('2024-09-02 10:18:11');
+        $this->inquiryFacade->create($inquiryData);
+
+        $inquiryData = $this->inquiryDataFactory->create();
+        $inquiryData->firstName = 'Ethan';
+        $inquiryData->lastName = 'Bailey';
+        $inquiryData->email = 'ethan.bailey@example.com';
+        $inquiryData->telephone = '777888222';
+        $inquiryData->companyName = null;
+        $inquiryData->companyNumber = null;
+        $inquiryData->companyTaxNumber = null;
+        $inquiryData->note = 'Do you have custom pricing for individuals?';
+        $inquiryData->product = $inquiryProduct;
+        $inquiryData->createdAt = new DateTimeImmutable('2024-09-02 17:42:37');
+        $this->inquiryFacade->create($inquiryData);
+
+        $inquiryData = $this->inquiryDataFactory->create();
+        $inquiryData->firstName = 'Madison';
+        $inquiryData->lastName = 'Perry';
+        $inquiryData->email = 'madison.perry@example.com';
+        $inquiryData->telephone = '888444555';
+        $inquiryData->companyName = 'Halcyon Holdings Corporation';
+        $inquiryData->companyNumber = '99887755';
+        $inquiryData->companyTaxNumber = 'US99887755';
+        $inquiryData->note = 'Can you provide more info?';
+        $inquiryData->product = $inquiryProduct;
```

Comment:
This is so true... I changed the logic a bit, so now it is possible to programmatically create an inquiry without the product – this may be useful in ERP connectors.
As a profit, I have set some inquiries to not have a product.  - ea2818aa53